homeassistant/components/hive/binary_sensor.py

```diff
@@ -67,6 +91,19 @@ async def async_setup_entry(
         True,
     )
 
+    devices = hive.session.deviceList.get("sensor")
+    if not devices:
+        return
+    async_add_entities(
```

Comment:
Please collect all entities in a list before adding them

Reply:
Completed